smart_contracts.rst

```diff
@@ -694,7 +694,7 @@ Must be enforced by the Raiden Client:
 .. Note::
     Any two consecutive balance proofs for ``P1``, named ``BP1k`` and ``BP1t`` were `k < t`,  must respect the following constraints:
 
-    1. A :term:`Direct Transfer` or a :term:`Mediated Transfer` with ``value`` tokens was finalized, therefore ``T1t == T1k + value`` and ``L1t == L1k``.
+    1. A :term:`Direct Transfer` or a :term:`Locked Transfer` with ``value`` tokens was finalized, therefore ``T1t == T1k + value`` and ``L1t == L1k``.
```

Comment:
What I wrote here contradicts the explanation you wrote above for `Locked Transfer`. I am here referring to the full, finalized transfer. You were defining only the `Locked Transfer` message.

I propose to keep both `Locked Transfer Message` concept and `Mediated Transfer`, where:
`Mediated Transfer` = the finalized transfer, composed of multiple Raiden client messages (`LockedTransfer`, `RevealSecret`, `Unlock` ...)
`Locked Transfer (Message)` = the first message that starts the mediated transfer, as defined above by you.

Reply:
Sounds good :)

Reply:
Yes, on the above line it should be `Mediated Transfer` , instead of `Locked Transfer`.